requirements.txt

```diff
@@ -2,9 +2,9 @@ black==21.12b0
 flake8==3.8.4
 iso639==0.1.4
 mypy==0.790
-pytest==6.1.1
+pytest==7.0.0
 requests-html==0.10.0
 requests==2.24.0
 segments==2.2.0
 setuptools==50.3.1
-unicodedataplus==13.0.0.post2
+unicodedataplus==14.0.0.post2
```

Comment:
This update is needed to resolve the Python 3.10 build failure at 1b85af2d5ad93e58fe3bff32df1fd89138e87479